test/functional/feature_asmap.py

```diff
@@ -42,8 +42,8 @@ def set_test_params(self):
         self.extra_args = [["-checkaddrman=1"]]  # Do addrman checks on all operations.
 
     def fill_addrman(self, node_id):
-        """Add 1 tried address to the addrman, followed by 1 new address."""
-        for addr, tried in [[0, True], [1, False]]:
+        """Add 2 tried address to the addrman, followed by 2 new address."""
+        for addr, tried in [[0, True], [1, True], [2, False], [3, False]]:
```

Comment:
In 964e1a8faec984df2799171bc2a9268a3765fbca:  Any specific reason to increase the number of added addresses here?

Reply:
Thanks, @0xB10C. An explanation in the commit message would be valuable.